Vagrantfile

```diff
@@ -72,6 +72,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     "rebuild_yum_cache" => false,
     "cpus"              => ENV['OPENSHIFT_NUM_CPUS'] || 2,
     "memory"            => ENV['OPENSHIFT_MEMORY'] || 2048,
+    "fixup_networking"  => true,
```

Comment:
If it is not required for custom images, should we not have an ENV var which may default to true? Or am I missing the logic?